frontend/src/components/Constants.js

```diff
@@ -1,9 +1,14 @@
 // change to actual values from database
 
 const Constants = {
-  consoles: [{ value: 'Wii' }, { value: 'iPhone' }, { value: 'XBOX' }],
-  ageRange: [{ value: '2 and under' }, { value: '2 to 5' }, { value: '5 and over' }],
-  ailments: [{ value: 'sad' }, { value: 'angry' }, { value: 'upset' }]
+  consoles: [
+    { value: 'XBox One' },
+    { value: 'Nintendo Switch' },
+    { value: 'PlayStation 4' },
+    { value: 'Nintendo 3DS' }
+  ],
+  ageRange: [{ value: 'Under 12' }, { value: '13 and Older' }],
```

Comment:
Change `Under 12` to `12 and Under`. It used to be Under 12 in the backend but that was fixed in master so make this change and pull master into your branch. Once you do these two things, everything should work but double check.